### ISSUE TITLE
[a11y-defect-2]: Provide default select option - Found in 508 Audit on form 21-4142

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#63590]
## Point of contact

@jeana-adhoc 

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

## Details

The 508 office returned an audit with this issue: When errors are automatically detected, the suggestion for fixing the error is given to the user.

On the `Step 3 of 12` screen, when you move the focus to “Country” field with a tab, assistive technology reads “Country (*Required) Combo box Required invalid entry”. See Figure 9 below.  (New: 08/02/2023) SME: Agree, the screen reader should not read the error until a selection has been made and the error is visible. <br /><br /><img width="460" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/fcb846d0-d77a-41f7-8f86-1b11beba2665">

## Reproduction steps
1. Turn on a screen reader, and navigate with a keyboard to the "country" field. The screen reader says it is an invalid entry, even though there is no visual error displaying on the screen.  We believe this is happening due to the inherit html `required` attribute on the select. Since nothing is selected, it is an invalid answer. 

## Proposed solution or next steps

1. Make `United States` the default answer in the drop down. Other instances of the address pattern makes the `United States` the first and default answer. If we could make this change for all of our 'simple forms', that would be fantastic.

## References, articles, or WCAG support

1. [WCAG Success criteria 3.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)

## Type of issue

- [X] Screenreader
- [X] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else
  - [X] Error messaging

**Comment by jeana-adhoc** [comment]
I know this is a v3 version, but this page actually handles it correctly as well: https://staging.va.gov/supporting-forms-for-claims/substitute-claimant-form-21P-0847/preparer-address, choosing `- Select -` would be a fine option as well.

**Comment by jeana-adhoc** [comment]
Checked on 7/19/24 - and this bug still exists in the form. I believe it's because it's not using our address pattern.
